GREEDY/DataManagers/DataManager.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using GREEDY.Models;
+using System.Windows.Forms;
+using System.IO;
+
+namespace GREEDY.DataManagers
+{
+    public class DataManager : IDataManager
+    {
+        private readonly SaveFileDialog _saveFileDialog;
+        private readonly DataGridView _getDataGridWiew;
+        private readonly string _saveDataDialogTitle = "Save an Image File";
+
+        public DataManager()
+        {
+            _saveFileDialog = new SaveFileDialog();
+            _getDataGridWiew = new DataGridView();
+        }
+
+        // not tested yet. Need to write a methods to create a file
+        public void SaveData(List<Item> itemList)
+        {
+            using (_saveFileDialog)
```

Comment:
using will dispose of the object, so calling this method a second time might throw an exception. If you use using new instance of an object should be created each time.

Reply:
Thanks for comment. This method not implemented yet so I will keep in mind